app/util/api/bamboo_clients.py

```diff
@@ -0,0 +1,65 @@
+from util.api.abstract_clients import RestClient
+
+BATCH_SIZE_SEARCH = 500
+
+
+class BambooClient(RestClient):
+
+    def get_build_plans(self, start=0, max_result=100):
```

Comment:
It feels like the purpose of this method is mixed.
If this is an API method to get plans without pagination, then params `start` and `max_results` make sense.
If this is an API method to get N plans, then `start` param does not needed.